packages/@lwc/compiler/src/transformers/__tests__/transform-javascript.spec.ts

```diff
@@ -51,3 +55,80 @@ it('should object spread', async () => {
     expect(code).toContain('b: 1');
     expect(code).not.toContain('...a');
 });
+
+it('should transform unforgeables when Lightning Web Security is on', async () => {
+    const actual = `
+        export const test = window.location.href;
+    `;
+    const { code } = await transform(actual, 'foo.js', {
+        ...TRANSFORMATION_OPTIONS,
+        enableLightningWebSecurityTransforms: true,
+    });
+
+    expect(code).toContain(
+        '(window === globalThis || window === document ? location : window.location).href'
+    );
+});
+
+it('should not transform unforgeables when Lightning Web Security is off', async () => {
+    const actual = `export const test = window.location.href;`;
+    const { code } = await transform(actual, 'foo.js', TRANSFORMATION_OPTIONS);
+    expect(stripWhitespace(code)).toMatch(stripWhitespace(actual));
+});
+
+it('should transform async await functions when Lightning Web Security is on', async () => {
+    const actual = `
+        export async function foo() {
+            await bar();
+        }
+    `;
+    const { code } = await transform(actual, 'foo.js', {
+        ...TRANSFORMATION_OPTIONS,
+        enableLightningWebSecurityTransforms: true,
+    });
+
+    expect(code).toContain('_asyncToGenerator');
+
+    expect(stripWhitespace(code)).toContain(
+        stripWhitespace(`
+        _foo = _asyncToGenerator(function* () {
+            yield bar();
+        });
+    `)
```

Comment:
☝️ This `expect` (starting on line 92) can be removed as the one on line 90 covers it.